lib/jsdom/living/nodes/Element-impl.js

```diff
@@ -430,6 +430,43 @@ class ElementImpl extends NodeImpl {
 
     return attr;
   }
+
+  insertAdjacentHTML(position, text) {
```

Comment:
Looks like "If context is null or a document, throw a DOMException with name "NoModificationAllowedError"." would be good to add a test for this too.